### ISSUE TITLE
Duplicate Form Field ID in Category Form Modal

### ISSUE THREAD

**Issue by medelm1** [medelm1/solar-sizer-calculator#4]
## Description:

When the "Add New Category" modal is open, the browser reports a warning about duplicate form field IDs. Ensure all form fields in the modal have unique IDs to prevent conflicts and accessibility issues.

**Comment by abbasDD** [comment]
Hi, I’d like to contribute and take this up. Is this issue open for community contributions, or do you prefer to keep it internal?

**Comment by medelm1** [comment]
@abbasDD Yes, this issue is open for community contributions. I’ll go ahead and assign it to you. Please feel free to share any updates or questions along the way. Looking forward to your PR!

**Comment by abbasDD** [comment]
Unable to track your mentioned issues

<img width="1363" height="686" alt="Image" src="https://github.com/user-attachments/assets/0b021cdd-15e5-41ce-a5f0-02b51665e120" />

maybe it was some library issues and I updated all libraries. 

Let me know if I am using wrong way to detect the issues.

I visited Manage Appliances from topbar and then click New Appliance. In new appliance form, I clicked Add New in category list to open "Add New Category" Modal

**Comment by abbasDD** [comment]
@medelm1
